<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InventoryManager</class>
 <widget class="QWidget" name="InventoryManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1091</width>
    <height>534</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Inventory Manager</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>291</width>
     <height>201</height>
    </rect>
   </property>
   <property name="title">
    <string>Inventory Manager</string>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>271</width>
      <height>141</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="addInventory">
       <property name="text">
        <string>Add Inventory</string>
       </property>
       <property name="autoDefault">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="performInvCycleCount">
       <property name="text">
        <string>Perform Inventory Cycle Count</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="createNewInventoryItem">
       <property name="text">
        <string>Create New Inventory Item</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="lowInvAlarmSettings">
       <property name="text">
        <string>Low Inventory Alarm Settings</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>10</y>
     <width>691</width>
     <height>211</height>
    </rect>
   </property>
   <property name="title">
    <string>Inventory List</string>
   </property>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>671</width>
      <height>181</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOn</enum>
    </property>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>125</number>
    </attribute>
    <row>
     <property name="text">
      <string>Item</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string>Part ID</string>
     </property>
     <property name="textAlignment">
      <set>AlignLeading|AlignVCenter</set>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Qty OnHand</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Part Description</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Colour</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Part Cost</string>
     </property>
    </column>
    <item row="0" column="0">
     <property name="text">
      <string>0001</string>
     </property>
     <property name="textAlignment">
      <set>AlignLeading|AlignVCenter</set>
     </property>
    </item>
    <item row="0" column="1">
     <property name="text">
      <string>4</string>
     </property>
    </item>
    <item row="0" column="2">
     <property name="text">
      <string>Back Case</string>
     </property>
    </item>
    <item row="0" column="3">
     <property name="text">
      <string>Black</string>
     </property>
    </item>
    <item row="0" column="4">
     <property name="text">
      <string>10</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>260</y>
     <width>691</width>
     <height>211</height>
    </rect>
   </property>
   <property name="title">
    <string>Low Inventory List</string>
   </property>
   <widget class="QTableWidget" name="tableWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>671</width>
      <height>181</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOn</enum>
    </property>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>125</number>
    </attribute>
    <row>
     <property name="text">
      <string>Item</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string>Part ID</string>
     </property>
     <property name="textAlignment">
      <set>AlignLeading|AlignVCenter</set>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Qty OnHand</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Part Description</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Colour</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Part Cost</string>
     </property>
    </column>
    <item row="0" column="0">
     <property name="text">
      <string>0001</string>
     </property>
     <property name="textAlignment">
      <set>AlignLeading|AlignVCenter</set>
     </property>
    </item>
    <item row="0" column="1">
     <property name="text">
      <string>4</string>
     </property>
    </item>
    <item row="0" column="2">
     <property name="text">
      <string>Back Case</string>
     </property>
    </item>
    <item row="0" column="3">
     <property name="text">
      <string>Black</string>
     </property>
    </item>
    <item row="0" column="4">
     <property name="text">
      <string>10</string>
     </property>
    </item>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
